masterdata/configuration/drugs/drug.csv

```diff
@@ -515,4 +515,36 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Vitamin A 50000 IU,Vitamin A,Oral capsule,50000 IU,
 ,,Vitamin A 100000 IU,Vitamin A,Oral capsule,100000 IU,
 ,,Vitamin A 100000 IU/ml,Vitamin A,Oral solution,100000 IU/ml,
-,,Anti-rabies vaccine,Anti-rabies vaccine,,,
\ No newline at end of file
+,,Anti-rabies vaccine,Anti-rabies vaccine,,,
+,,Benzoic Acid+Salicylic Acid Ointment 6%w/w+3%w/w 1x30g,Benzoic acid / salicylic acid,Ointment,6%w/w+3%w/w,
+,,Bromhexine Syrup 4mg/5ml 1x100ml,Bromhexine,Syrup,4mg/5ml,
+,,Diclofenac Topical Form 1% 1x30gm,Diclofenac,Gel,1%,
+,,Paracetamol Drops 100mg/ml 1x15ml,Acetaminophen,Drops,100mg/ml,
+,,Paracetamol Syrup 125mg/5ml 1x60ml,Acetaminophen,Syrup,125mg/5ml,
+,,Paracetamol Syrup 250mg/5ml 1x60ml,Acetaminophen,Syrup,250mg/5ml,
+,,Silver Sulphadiazine Cream 125g 1x125g,Silver sulfadiazine,Cream,125g,
+,,Sodium Chloride (Nasal) Drops 0.65%w/v 1x15ml,Sodium chloride,Drops,0.65%w/v,
+,,Vitamin C Chewable (Ascorbic Acid) Tablet 500mg 1x10x10,Ascorbic acid,Tablet,500mg,
+,,Benzoin Compound Tincture As per IP 1x500ml,Benzoin compound,Tincture,,
+,,Benzyl benzoate Lotion 25% 1x100ml,Benzyl benzoate,Lotion,25%,
+,,Liquid Paraffin Solution 60ml 1x60ml,Paraffin,Solution,60ml,
+,,Povidone Iodine Ointment 5% 1x125g,Povidone-iodine,Ointment,5%,
+,,Povidone Iodine Ointment 5% 1x15g,Povidone-iodine,Ointment,5%,
+,,Amoxicillin dry powder,Amoxicillin,Powder,1x10,
+,,Amoxicillin tab dispersible,Amoxicillin,Tablet,1x10,
+,,"Antacid | suspension | Activated diamithecon 50mg, Mg(oh)2 250mg, dried A(OH)3 250mg, MG, Al silicate 50mg tablet",Dimethicone,Suspension,1x200ml,
+,,Atenolol Tablet 25mg 1x14x10,Atenolol,Tablet,25mg,
+,,Clotrimazole 400 mg tablet,Clotrimazole,Tablet,400mg,
+,,Dicyclomine Hydrochloride Drops 10mg/5ml 1x10ml,Dicyclomine hydrochloride,Drops,10mg/5ml,
+,,Domperidone Syrup 1mg/ml 1x30ml,Domperidone,Syrup,1mg/ml,
+,,Ferrous sulphate tablet 100 mg iron element & folic acid .5 mg,Ferrous sulfate,dried,Tablet,,
+,,Methyl rosanilium chloride/Aqueous solution, Gentian violet,Solution,.05%,
+,,Salbutamol sulphate Syrup 2mg/5ml 1x100ml,Albuterol sulfate,Syrup,2mg/5ml,
+,,Salbutamol sulphate tablet 2mg,Albuterol sulfate,Tablet,2mg,
+,,Salbutamol sulphate tablet 4mg 1x10x10,Albuterol sulfate,Tablet,4mg,
+,,Theophylline +Etophylline syrup 23+77 mg,Etofylline,Syrup,23+77 mg,
+,,Theophylline +Etophylline tablet 23+77 mg,Theophylline,Tablet,23+77 mg,
```

Comment:
This isn't likely correct. Its a combination drug, but mapped to a single concept. Should be mapped to a combination concept. Please validate with akhil/angshu.

Reply:
Sure, will check with them

Reply:
@gsluthra The combination concept isn't available in CIEL. So created a combination concept (Etophylline / Theophylline) and updated the code.